### ISSUE TITLE
WhatsApp

### ISSUE THREAD

**Issue by mcleinn** [ushahidi/pingapp#113]
Hello, I'm interested in WhatsApp support for Ushahidi. I saw in the presentation that this messenger is supported, however, on a quick check I didn't find it in the source. Did I oversee this code, or is the plugin still on the ToDo list? Thank you

**Comment by rjmackay** [comment]
ping @dkobia is this planned for Ping?
We haven't planned this for Ushahidi V3, but provider code written for Ping may be reusable.

**Comment by dkobia** [comment]
@rjmackay Yes it is planed, but WhatsApp has an elusive system, no _official_ API, and some _unofficial_ ones so there are no guarantees. We will however try.

**Comment by mcleinn** [comment]
Thank you for the information. While you are at it, you might try Viber as
well. At the very least there are "spy" apps, which claim to read Viber
messages on rooted Androids.

Regards,
Tobias

2014-01-28 David Kobia notifications@github.com
